### PR TITLE
feat: implemented new branding page ui

### PR DESCRIPTION
**Pull request by BiswaViraj**
### What change does this PR introduce?
- Updated the Brand page UI
- Added a space prop for Card
- Removed padding on table element
- Updated the add new layout action button

### Other information (Screenshots)

image

image

image

image

### CODE REVIEW

File: apps/web/src/pages/brand/tabs/LayoutsListPage.tsx [hunk 1]
```diff
@@ -170,9 +170,12 @@ export function LayoutsListPage({ handleLayoutAnalytics }: LayoutsListPageProps)
...
           >
-            <Text gradient={!readonly} color={colors.B60}>
-              + Create New Layout
-            </Text>
+            <Group spacing={8}>
+              <PlusFilled width={24} height={24} />
+              <Text gradient={!readonly} color={colors.B60} weight="bold">
+                Add New Layout
+              </Text>
+            </Group>
```
Comment (LetItRock) [inline apps/web/src/pages/brand/tabs/LayoutsListPage.tsx:178]: there should be already a button created for this on the new Integrations List page, can we reuse it and move to the components folder?

Comment (BiswaViraj) [reply apps/web/src/pages/brand/tabs/LayoutsListPage.tsx]: Created the reusable component

**Comment by LetItRock** [comment]
hey @BiswaViraj, can you please take a look into these visual problems?

Screenshot 2023-08-04 at 11 08 05

**Comment by BiswaViraj** [comment]
> hey @BiswaViraj, can you please take a look into these visual problems?
> 
> Screenshot 2023-08-04 at 11 08 05

@LetItRock , the table header, content wont align with the above contents if I add padding.
other way will be to make table full width, and replace borders with pseudo element having width: 100% - paddingLeft+ paddingright

image

**Comment by LetItRock** [comment]
> @LetItRock , the table header, content wont align with the above contents if I add padding. other way will be to make table full width, and replace borders with pseudo element having width: 100% - paddingLeft+ paddingright

you can do the trick with a negative margin on the table wrapper like add the attribute to that wrapper like a `data-table-holder` and then target it in the CSS, I did something similar in my PR ref: https://github.com/novuhq/novu/blob/47b6c20df92be2bf1dbd5fba70da756c78d6d47c/apps/web/src/design-system/table/Table.tsx#L180:

Screenshot 2023-08-04 at 14 35 39

**Comment by BiswaViraj** [comment]
image

updated the layout

**Review by LetItRock — APPROVED** [review]

**Pull request merged**